src/vaadin-select-text-field.html

```diff
@@ -27,27 +26,13 @@
         return 'vaadin-select-text-field';
       }
 
-      static get template() {
```

Comment:
in order to avoid breaking change and make the fix possible to release as 2.0.3 (and backwards compatible with V12), please add a check for `window.Vaadin.TextFieldElement.version`:
- for text-field `2.1`, keep the old template getter behavior.
- for text-field `2.3` and above, just return `super.template`

For the V14 LTS, we should bump a new minor and drop the conditional check.

Reply:
Updated. Detecting slot existence now.